pinot-core/src/main/java/org/apache/pinot/core/common/DataFetcher.java

```diff
@@ -419,6 +419,18 @@ public void fetchNumValues(String column, int[] inDocIds, int length, int[] outN
     _columnValueReaderMap.get(column).readNumValuesMV(inDocIds, length, outNumValues);
   }
 
+  /**
+   * Fetch the bytes values for a multi-valued column.
+   *
+   * @param column Column name
+   * @param inDocIds Input document Ids buffer
+   * @param length Number of input document Ids
+   * @param outValues Buffer for output
+   */
+  public void fetchBytesValues(String column, int[] inDocIds, int length, byte[][][] outValues) {
```

Comment:
(minor) Move it in front of `fetchNumValues()`

Reply:
Done